frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -137,7 +137,10 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
 
   return (
     <>
-      <div ref={contentRef} className="odc-pipeline-builder-form">
+      <div
+        ref={contentRef}
+        className="odc-pipeline-builder-form ocs-quick-search-modal__no-backdrop"
```

Comment:
Instead of having this classname `ocs-quick-search-modal__no-backdrop`  in PipelineBuilderForm.tsx and TopologyView.tsx, can we have this directly in the quickSearchModal.tsx. Maybe a configuration prop (**backdropEnabled**) that decides if the backdrop classname has to be added or not (default can be a false). WDYT?

quickSearchModal.tsx
```
<Modal
className={cx('ocs-quick-search-modal', {
'ocs-quick-search-modal__no-backdrop': !backdropEnabled})} // defaults to false
...
>

Reply:
I don't think this is needed, [from my comment](https://github.com/openshift/console/pull/9880#pullrequestreview-741643777):

> I'm adding LGTM as I don't think @karthikjeeyar's suggestion about classNames usage will add much if we need to return to it.

Reply:
Yeah, we can skip it.